wpimath/src/main/native/include/frc/controller/LinearPlantInversionFeedforward.h

```diff
@@ -7,7 +7,15 @@
 #include <array>
 #include <functional>
 
+// FIXME: GCC 12.1 gives false positive
+#if __GNUC_MAJOR__ >= 12
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Warray-bounds"
+#endif  // __GNUC_MAJOR__ >= 12
```

Comment:
Seems like this should go in as a patch to Eigen, otherwise users will get false positives too?